### DISCUSSION TITLE
Quarto - Code Annotation

### DISCUSSION THREAD

**Discussion by giscus** [quarto-dev/quarto-cli#11410] [Blog]
# Quarto - Code Annotation

In Quarto 1.3, you can add line based annotations to code chunks to highlight or explain parts of your code.

https://quarto.org/docs/blog/posts/2023-03-13-code-annotation/

**Comment by factorialmap** [comment]
Awesome. Thank you so much !

**Comment by treysaddler** [comment]
Very useful

**Comment by be-favorite** [comment]
That looks really helpful. Thanks!

**Comment by sebastian-c** [comment]
This is really nice, but is there an option to show comments on hover for HTML? Or to highlight associated lines on hover?

**Reply by dragonstyle** [reply]
Yes! More here: https://quarto.org/docs/authoring/code-annotation.html#html-styles

**Comment by rungdung** [comment]
Is there an attempt to tie this into a larger semantic annotation experience/standard as of now?
